docs/source/user_guide/installation/index.rst

```diff
@@ -19,10 +19,6 @@ It can be installed using ``pip`` or ``conda``, or from source.
 
    The ``pybamm`` package on conda-forge installs PyBaMM with all the `required and optional dependencies <#dependencies>`_ available on conda-forge.
```

Comment:
The required dependencies section should be updated to include `pybammsolvers`.

Reply:
Ok I will fix this

Reply:
Done. Maintaining that list seems like a bit of wasted effort. We should probably just link people to our pyproject.toml file or something. Nobody needs to manually install that list of dependencies, they are just installed by pip or conda